src/main/resources/db/migration/V1_132__print_files_to_clob_update.sql

```diff
@@ -0,0 +1,216 @@
+DROP PROCEDURE juror_mod.printfiles_to_clob(int4);
+
+CREATE OR REPLACE PROCEDURE juror_mod.printfiles_to_clob(IN p_document_limit integer DEFAULT 1000000)
+ LANGUAGE plpgsql
+AS $procedure$
+DECLARE
+	v_form_type VARCHAR(6);
+	v_max_rec_len INTEGER;
+	v_ext_date DATE;
+	forms RECORD;
+	v_count INTEGER:=0;
+
+BEGIN
+	v_ext_date :=	CASE
+						WHEN TO_CHAR(NOW(),'SSSSS')::INTEGER < 64800
+              				THEN NOW()::date - 1
+               				ELSE NOW()
+              		END;
+
+    -- delete redundant letter requests
+	CALL juror_mod.delete_printfiles();
+
+	FOR forms IN
+		SELECT	fr.form_type,
+				fr.max_rec_len::integer
```

Comment:
We previously discussed that the name of t_form_attr.max_rec_len is misleading. it's not a max length, it's just the length.

How about using an alias of rec_len for max_rec_len ?

If we do that, we should also name are subsequent parameters/variable to be rec_len not max_rec_len.

Reply:
Added alias